<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeraChemInputDialog</class>
 <widget class="QDialog" name="TeraChemInputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>471</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TeraChem Input</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Title:</string>
       </property>
       <property name="buddy">
        <cstring>titleLine</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="titleLine">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Calculation:</string>
       </property>
       <property name="buddy">
        <cstring>calculationCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="calculationCombo">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>Single Point Energy</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gradient</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Geometry Optimization</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>27</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Theory:</string>
       </property>
       <property name="buddy">
        <cstring>theoryCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="theoryCombo">
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>HF</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>BLYP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B3LYP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B3LYP1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B3LYP5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PBE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>REVPBE</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkUnrestricted">
         <property name="text">
          <string>Unrestricted</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Charge:</string>
       </property>
       <property name="buddy">
        <cstring>chargeSpin</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QSpinBox" name="chargeSpin">
         <property name="minimum">
          <number>-99</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>58</width>
           <height>27</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Coordiantes</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QComboBox" name="comboCoord">
         <item>
          <property name="text">
           <string>PBD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>XYZ</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Multiplicity:</string>
       </property>
       <property name="buddy">
        <cstring>multiplicitySpin</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Basis:</string>
       </property>
       <property name="buddy">
        <cstring>basisCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QComboBox" name="basisCombo">
         <property name="accessibleDescription">
          <string notr="true"/>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>STO-3G</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3-21G</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6-31G(d)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6-31G(d,p)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6-31+G(d)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6-311G(d)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cc-pVDZ</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>58</width>
           <height>26</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="5" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>26</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="moreButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Hide Preview</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="3">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QSpinBox" name="multiplicitySpin">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="previewText">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="enableFormButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Use Form</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>48</width>
         <height>26</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="generateButton">
       <property name="text">
        <string>Generate...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>titleLine</tabstop>
  <tabstop>calculationCombo</tabstop>
  <tabstop>theoryCombo</tabstop>
  <tabstop>basisCombo</tabstop>
  <tabstop>chargeSpin</tabstop>
  <tabstop>moreButton</tabstop>
  <tabstop>previewText</tabstop>
  <tabstop>generateButton</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>resetButton</tabstop>
  <tabstop>enableFormButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>TeraChemInputDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
